optimism.tokenlist.json

```diff
@@ -891,7 +891,40 @@
       "extensions": {
         "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
       }
-	}
+	},
+    {
+      "chainId": 1,
+      "address": "0x3af33bef05c2dcb3c7288b77fe1c8d2aeba4d789",
+      "name": "Kromatika",
+      "symbol": "KROM",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/KROM.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1"
+      }
+    },
+    {
+      "chainId": 10,
+      "address": "0xf98dcd95217e15e05d8638da4c91125e59590b07",
+      "name": "Kromatika",
+      "symbol": "KROM",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/KROM.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 69,
+      "address": "0x0f747ed5De34aaDc17E39368b7d90da2D0545319",
```

Comment:
Can you verify this in Etherscan pls

Reply:
Also can you send me some tokens on kovan I can test with at 0xe3a90926133fef3c58514fA1aAeF75E0998aedD5

Reply:
https://kovan.etherscan.io/tx/0x4510d14647d611e2eb883c6a57e61ef85b6b97d5ceead79f7ae5cd016ec43c5c

Reply:
sent 1 mil KROM on kovan

Reply:
> 

This was a wrong contract, the new one on optimism-kovan is verified.
https://kovan-optimistic.etherscan.io/address/0x9e1028F5F1D5eDE59748FFceE5532509976840E0#code